src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -3075,8 +3089,10 @@ public static void M2 (this ref readonly int receiver)
 """;
         var comp = CreateCompilation(src);
 
-        // PROTOTYPE: A similar warning is expected for 'GetInt().M1()';
         comp.VerifyEmitDiagnostics(
+            // (5,9): warning CS9193: Argument 0 should be a variable because it is passed to a 'ref readonly' parameter
```

Comment:
>Argument 0 [](http://example.com/codeflow?start=38&length=10)

"Argument 0" part could be somewhat confusing. A PROTOTYPE comment?

Reply:
Did you add a PROTOTYPE comment in implementation? Consider adding it here as well, because it is clearer what the concern is about.